src/main/java/frc/robot/subsystems/shooter/commands/Shoot.java

```diff
@@ -1,15 +1,33 @@
 package frc.robot.subsystems.shooter.commands;
 
-import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
-
-public class Shoot extends SequentialCommandGroup {
-    public Shoot(double required_velocity) {
-        /*
-        Once the rest of the robot is operational, another command will
-        be added to feed the balls to the shooter.
-         */
-        addCommands(
-                new PrepareShooter(required_velocity)
-        );
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.shooter.Shooter;
+
+import java.util.function.DoubleSupplier;
+
+public class Shoot extends CommandBase {
+    private final Shooter shooter;
+    private final DoubleSupplier velocitySetpoint;
+
+    /**
+     * Constructor.
+     *
+     * @param shooter          is the shooter subsystem.
+     * @param velocitySetpoint is the setpoint for the shooter wheel. [rps]
+     */
+    public Shoot(Shooter shooter, DoubleSupplier velocitySetpoint) {
+        this.shooter = shooter;
+        this.velocitySetpoint = velocitySetpoint;
+        addRequirements(shooter);
+    }
+
+    @Override
+    public void execute() {
+        shooter.setVelocity(Shooter.getSetpointVelocity(velocitySetpoint.getAsDouble()));
```

Comment:
You don't need to use the function `getSetpointVelocity`, it should be passed in the supplier (look at Outtake for example)